src/Tribe/Main.php

```diff
@@ -1126,4 +1126,21 @@ public function register_plugin_autoload_paths() {
 
 		$this->get_autoloader_instance()->register_prefixes( $prefixes );
 	}
+
+	/**
+	 * Get the component template.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template
+	 */
+	public function get_component_template(): Tribe__Template {
```

Comment:
I'm wondering if we should use the word components in plural, given that we're gonna be calling the components folder. Food for thought, I'm good with either way.